src/background/actions/assessment-actions.ts

```diff
@@ -51,4 +52,5 @@ export class AssessmentActions {
     public readonly continuePreviousAssessment = new AsyncAction<number>();
     public readonly loadAssessment = new AsyncAction<LoadAssessmentPayload>();
     public readonly updateDetailsViewId = new AsyncAction<OnDetailsViewInitializedPayload>();
+    public readonly transferAssessment = new AsyncAction<TransferAssessmentPayload>();
```

Comment:
It feels a little odd to put this in `assessmentActions` but that might just be a naming thing. Because other actions are operations on the current assessment, `transferAssessment` sounds like we're sending our current assessment's data somewhere else. Something like `importAssessmentData` might more readily describe the action.

Reply:
Hm, I was thinking about this as well. I was going with transfer because it relates specifically to the `DataTransfer` prefix (as far as store/actions/etc.) because otherwise it feels like it's a synonym for loadAssessment.

I'm going to change it to `loadAssessmentFromTransfer', let me know what you think.